### ISSUE TITLE
ResultJson.fromJson can not handle arrays 

### ISSUE THREAD

**Issue by hyyan** [BasisHub/components#153]
The defined method  `static ResultSet -> fromJson(String js)` cannot parse correctly JSON data which contain arrays.

**Json Data**
```
[
  {
    "orgHierarchy": [
      "Erica Rogers",
      "Malcolm Barrett",
      "Esther Baker",
      "Brittany Hanson"
    ],
    "jobTitle": "Fleet Coordinator",
    "employmentType": "Permanent"
  }
]
```

**In BBj program**
```java
data! = "my json data"
?  ResultSet.fromJson(data!)
```

**Output of the first row** 

```js
[
  {
    "orgHierarchy": "[Erica Rogers, Malcolm Barrett, Esther Baker, Brittany Hanson]",
    "jobTitle": "Fleet Coordinator",
    "employmentType": "Permanent"
  }
]
```

Expected output : 
```js
[
  {
    "orgHierarchy": "[\"Erica Rogers\",\"Malcolm Barrett\",\"Esther Baker\",\"Brittany Hanson\"]",
    "jobTitle": "Fleet Coordinator",
    "employmentType": "Permanent"
  }
]
```

**Comment by Frank-en-stein** [comment]
I'm planning to look into this issue and need a little help. Could you provide me with a possible meta-data or some instruction on how to make meta-data for the test JSON given above?

**Comment by Frank-en-stein** [comment]
Hi! Just to be sure, is the expected value of key "orgHierarchy" a string and not an array?

**Comment by DaAmore** [comment]
The expected Value is array, but it seems to be parsed as String.

**Comment by hyyan** [comment]
@Frank-en-stein @DaAmore Seems to be fixed in BBj 19. would be great if we can port the fix to BBj 18.

The expected output is a stringified JSON array for `orgHierarchy`. Here a BBj demo you can use:
```java
str$ = "[{""orgHierarchy"":[""Erica Rogers"",""Malcolm Barrett"",""Esther Baker"",""Brittany Hanson""],""jobTitle"":""Fleet Coordinator"",""employmentType"":""Permanent""}]"

? ResultSet.fromJson(str$).toJson()
```

**Comment by Frank-en-stein** [comment]
Fixing it on the branch BBJ1821GA should suffice, right?

**Comment by StephanWald** [comment]
resolved by 19.2x (latest)
